### PR TITLE
Sync video classification pipeline with huggingface_hub spec

### PR DESCRIPTION
**Pull request by Rocketknight1**

### CODE REVIEW

**Comment by Rocketknight1** [comment]
cc @LysandreJik @Wauplin - this was a simple one, no Hub changes needed!

File: src/transformers/pipelines/video_classification.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import warnings
```
Comment (LysandreJik) [inline src/transformers/pipelines/video_classification.py:1]: This one seems to be lacking a disclaimer; can you add it?

**Review by LysandreJik — APPROVED** [review]
Thank you!

**Review by Wauplin — APPROVED** [review]
:fire:

**Pull request merged**